debian/copyright

```diff
@@ -0,0 +1,37 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: libcs50
+Source: http://github.com/cs50/libcs50/releases
+
+Files: *
+Copyright: 2017 David J. Malan <malan@harvard.edu>
+           2017 Glenn Holloway <holloway@eecs.harvard.edu>
+License: BSD-3-Clause
+
+Files: debian/*
```

Comment:
That was a template generated by `dh_make` actually. Thought to refer everything related to packaging to sysadmins. Can remove though.